src/main/java/frc/robot/commands/drive/DriveAndAim.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.drive;
+
+import frc.robot.Constants;
+import frc.robot.Vision;
+import frc.robot.subsystems.DriveSubsystem;
+import friarLib2.utility.Vector3309;
+
+/**
+ * Point at the target as the robot moves about the field
+ */
+public class DriveAndAim extends FieldRelativeTeleopControl {
+    public DriveAndAim (DriveSubsystem drive) {
+        super(drive);
+    }
+
+    /**
+     * Use the data from the vision system to point towards the target
+     */
+    @Override
+    protected double calculateRotationalSpeed (Vector3309 translationalSpeeds) {
```

Comment:
since you're returning in the if statement, you don't need an else. Code becomes much more readable the less you can use elses.

e.g., I'd write this as 

```
protected double calculateRotationalSpeed (Vector3309 translationalSpeeds)
{
    if (Vision.shooterCamera.hasTargets())
    {
        return Constants.Drive.VISION_AIM_PID.calculate(Vision.shooterCamera.getBestTarget().getX());
    }
    
    // Use driver input if no target is found
    return super.calculateRotationalSpeed(translationalSpeeds);
}
```